README.md

```diff
@@ -141,13 +141,30 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                      **[Soham Shah](https://www.linkedin.com/in/soham-shah-a97b5917b/)**                                                                                       |
+| <a href="https://www.linkedin.com/in/soham-shah-a97b5917b/"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
+
+|                                     <a href="https://github.com/seema1711"><img src="https://avatars.githubusercontent.com/u/48756444?s=400&u=eb84811c137fad97d5d02c2cb99c77bc39676a26&v=4" width=150px height=150px /></a>                                      |
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                      **[Seema Sharan](https://www.linkedin.com/in/seemasaharan/)**                                                                                       |
+| <a href="https://www.linkedin.com/in/seemasaharan/"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
+
+|                                     <a href="https://github.com/vybhav72954"><img src="https://avatars.githubusercontent.com/u/49750343?s=400&u=191d8b88762720b0d229531fb1118e9715f0489b&v=4" width=150px height=150px /></a>                                      |
+| :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+|                                                                                      **[Vybhav Chaturvedi](https://www.linkedin.com/in/vybhav-chaturvedi-0ba82614a/)**                                                                                       |
+| <a href="https://www.linkedin.com/in/vybhav-chaturvedi-0ba82614a/"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
```

Comment:
They look stacked vertically. Can you make them look stacked horizontally for better readability?